private-templates-service/client/src/components/Dashboard/PreviewModal/PreviewModal.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
On a small screen, the left sidebar disappears and shows the recent/drafts
![image](https://user-images.githubusercontent.com/7745961/73902261-0a8cc980-484a-11ea-9f9f-38c207cd400f.png)

Reply:
We don't actually have mobile designs for this page yet so I won't do too much work on it. I just modified the margins so that it spans the whole page.

![image](https://user-images.githubusercontent.com/19780117/73982996-40818a80-48ea-11ea-9a29-84fc3f1a7a93.png)